### PR TITLE
Add Fig features

### PR DESCRIPTION
**Pull request by grant0417**

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -208,4 +208,7 @@
   contact: https://github.com/msclock/features/issues
   repository: https://github.com/msclock/features
   ociReference: ghcr.io/msclock/features
-  
+- name: Fig Dev Container Features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:211]: Hi 👋 

Do you mind adding the missing `maintainer` input?

File: _data/collection-index.yml [hunk 2]
```diff
@@ -208,4 +208,7 @@
...
+  contact: https://github.com/withfig/features/issues
+  repository: https://github.com/withfig/features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:214]: Looks like the `features` package is set to private. If you'd like the supporting tool to show them, would you switch it to public? 

image

**Comment by grant0417** [comment]
Hey @samruddhikhandale, resolved the issues!

**Review by samruddhikhandale — APPROVED** [review]
Great, looks good. thanks!

**Pull request merged**